front-end/src/components/Post/GovernanceSideBar/Proposals/ProposalVoteInfo.tsx

```diff
@@ -42,6 +46,7 @@ const ProposalVoteInfo = ({ className, proposalId }:  Props) => {
 					setDeposit(formatBnBalance(proposal.balance, { numberAfterComma: 2, withUnit: true }));
 				}
 			});
+			setLoadingStatus({ isLoading: false, message: '' });
```

Comment:
Following up on the previous comment, here, we set the `isLoading` to false, so the `apiReady` is true.